src/test/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSourceUnitTest.java

```diff
@@ -33,14 +36,13 @@
     @DataProvider(name = "loadReads")
     public Object[][] loadReads() {
         return new Object[][]{
-                {NA12878_chr17_1k_BAM, null},
+                {NA12878_chr17_1k_BAM, null, ValidationStringency.SILENT},
 
-                {dirBQSR + "HiSeq.1mb.1RG.2k_lines.alternate.bam",  null},
-                {dirBQSR + "expected.HiSeq.1mb.1RG.2k_lines.alternate.recalibrated.DIQ.bam", null},
+                {dirBQSR + "HiSeq.1mb.1RG.2k_lines.alternate.bam", null, ValidationStringency.STRICT},
+                {dirBQSR + "expected.HiSeq.1mb.1RG.2k_lines.alternate.recalibrated.DIQ.bam", null, ValidationStringency.STRICT},
```

Comment:
Is there a reason why some test cases are using `SILENT` and others are using `STRICT`?